nginx/Dockerfile

```diff
@@ -1,4 +1,17 @@
+FROM nginx:1.25-alpine as build
+LABEL org.opencontainers.image.source https://github.com/CDCgov/prime-simplereport
+
+WORKDIR certs
+
+RUN apk add mkcert --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing/
```

Comment:
The `testing` branch of the `edge` repo can be unstable. `mkcert` could disappear from here at any time. When it does, we'll be dead in the water until we can get this updated. Can we modify this to use one of the more stable installation methods for mkcert, at least until this package makes it into the stable `alpine` builds?

Reply:
yeah, I am open to input here, there doesn't seems to be an easy way to install `mkcert` on linux (or windows) outside installing brew and or compiling from source.

I anticipate this image will rarely be changing, thus after initial build, the image will be cached in the github registry and we wont be building it that often, unless something changes in the nginx build directory, and if that package is removed once we rebuild, then the PR will fail, but not main, since the "latest" image will still be in github registry

How do you feel about just shipping the generated certs with the source code, rather than generating it everytime? and provide a script + docs to generate again manually if needed

My goal is to make this easy to run locally, and previously we didnt have a good way to generate this locally because it was done in the CI, which makes things kinda hard to figure out why things weren't working locally.

Reply:
I'm not keen on committing those certs. An alternative to what you have on line `6` could be:

```
RUN curl -JLO "https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-linux-amd64"
RUN chmod +x mkcert-v1.4.4-linux-amd64
RUN cp mkcert-v1.4.4-linux-amd64 /usr/local/bin/mkcert
```

This has the benefit of letting up pin a version.

Or if we don't care about pinning the version, we could:

```
RUN curl -JLO "https://dl.filippo.io/mkcert/latest?for=linux/amd64"
RUN chmod +x mkcert-v*-linux-amd64
RUN cp mkcert-v*-linux-amd64 /usr/local/bin/mkcert
```

A small amount of testing was done to validate this comment so you'll have to double-check. Would this address the core of your concern, @rin-skylight ?

Reply:
I went ahead an added the pre-built binary option @alismx suggested and it works flawlessly in the CI and also on m1 macs with Rosetta. Let me know if you still have any concerns @rin-skylight